lms/djangoapps/grades/new/course_grade.py

```diff
@@ -114,7 +111,7 @@ def summary(self):
         grade_summary['totaled_scores'] = self.subsection_grade_totals_by_format
         grade_summary['raw_scores'] = list(self.locations_to_weighted_scores.itervalues())
 
-        self._log_event(log.warning, u"grade_summary, percent: {0}, grade: {1}".format(self.percent, self.letter_grade))
+        self._log_event(log.info, u"grade_summary, percent: {0}, grade: {1}".format(self.percent, self.letter_grade))
```

Comment:
So right now we're logging every time this property is accessed - would it be more useful to log when the backing lazy property (`self.grade_value`) is calculated?